packages/itwin/tree-widget/src/test/trees/category-tree/CategoriesTreeComponent.test.tsx

```diff
@@ -27,31 +40,215 @@ describe("<CategoriesTreeComponent />", () => {
     await IModelApp.shutdown();
   });
 
+  beforeEach(async () => {
+    vpMock = mockViewport();
+    const { presentationManager } = mockPresentationManager();
+    sinon.stub(Presentation, "presentation").get(() => presentationManager.object);
+    await Presentation.initialize();
+  });
+
+  afterEach(() => {
+    imodelMock.reset();
+    sinon.restore();
+    Presentation.terminate();
+  });
+
   const imodelMock = moq.Mock.ofType<IModelConnection>();
+  const viewsMock = moq.Mock.ofType<IModelConnection.Views>();
+
+  imodelMock.setup((x) => x.views).returns(() => viewsMock.object);
```

Comment:
Looks like this should be done in `beforeEach`

Reply:
fixed